cgroup2/manager.go

```diff
@@ -763,37 +755,42 @@ func (c *Manager) EventChan() (<-chan Event, <-chan error) {
 	return ec, errCh
 }
 
-func parseMemoryEvents(out map[string]interface{}) (Event, error) {
+func parseMemoryEvents(out map[string]string) (Event, error) {
 	e := Event{}
 	if v, ok := out["high"]; ok {
-		e.High, ok = v.(uint64)
-		if !ok {
+		val, err := parseUint(v, 10, 64)
+		if err != nil {
 			return Event{}, fmt.Errorf("cannot convert high to uint64: %+v", v)
```

Comment:
Not related to this PR, but it would be easier to return `nil`not `Event{}` every time an error is returned

Reply:
Agree, I'm going to leave though to avoid too much refactor off the main goal. We can always do a followup if need be